addon/styles/_frost-guage.scss

```diff
@@ -0,0 +1,33 @@
+.frost-guage {
+  position: relative;
+
+  &-text {
```

Comment:
@sglanzer Really? I've always use `&-text` syntax. For the same reason I use `(concat css '-text')` in the template. It makes it much less painful when prefixing all classes with the component name. Is there some other way to facilitate finding the definition of a rule? Sass source mapping or something? While I'm all for making it easier to find the definition for styles applied to a class, I'm also a fan of making life easier for developers, and making everyone write out the full class name seems unnecessarily painful. Would you also suggest not using the `(concat css '-text')` pattern in the templates for the same reason?